README.md

```diff
@@ -111,10 +111,9 @@ Workflows require few configuration options defined as variables and secrets:
 
 - Variable `EDS_DOMAIN_URL`: a URL to Edge Delivery Services domain,
   e.g.: `https://main--puresight-demo--websight-rnd.hlx.live`.
-- Variable `STREAMX_KEY_PREFIX` (optional): a prefix to add to each publication key,
-  e.g.: `/published/puresight`.
+- Variable `STREAMX_KEY_PREFIX` (optional): a prefix to add to each publication key.
```

Comment:
you removed usage of prefix. Is it used somewhere else ?

Reply:
it was used only in the GH actions scriot to publish/unpublish. But we don't need this anymore